worker/handle-message.ts

```diff
@@ -47,6 +48,19 @@ export const testUtils = {
 	getCurrentView: (): string | undefined => view
 };
 
+const sortObjectArrayByName = (obj: Array<Folder>): Array<Folder> =>
```

Comment:
I think the function name is too generic. To be more clear it should mention the folder (like _sortFoldersByName_).
Also a comment should mention the fact that no copy is made and the provided array is changed.

Reply:
I agree about the name which is too generic. I don't know about the comment as everything inside this file mutates the provided object.
I think it is still fine as it is only used inside the file and it is not exported.
If this will be exported and used somewhere else we could consider to rename it and also add some warning comments about the usage